src/Test.UnitTests.Sarif.Driver/Test.UnitTests.Sarif.Driver.csproj

```diff
@@ -31,18 +31,18 @@
   </Target>
 
   <ItemGroup>
-    <PackageReference Include="FluentAssertions" Version="5.10.2" />
-    <PackageReference Include="Microsoft.Coyote" Version="$(CoyoteVersion)" />
-    <PackageReference Include="Microsoft.Coyote.Test" Version="$(CoyoteVersion)" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.0-preview-20220707-01" />
-    <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="FluentAssertions" Version="6.8.0" />
+    <PackageReference Include="Microsoft.Coyote" Version="1.7.3" />
```

Comment:
@michaelcfanning Why hard code the version here, rather than use `CoyoteVersion` variable?

Reply:
@shaopeng-gh If you are updating the PR, can you make this change as well? Otherwise I can make the change.

Reply:
Oops. :)

Reply:
I have updated with the `$(CoyoteVersion)`

Reply:
Nice work!